### ISSUE TITLE
vrscene creator in Maya and Deadline URL

### ISSUE THREAD

**Issue by antirotor** [pypeclub/OpenPype#2578]
## Bug

`vrscene` Creator plugin in Maya is still using old way to determine Deadline webservice URL. As this is no longer set, creator is failing.

[cuID:23dqgyq]

**Comment by BigRoy** [comment]
The code in Maya's [`CreateVRayScene`](https://github.com/pypeclub/OpenPype/blob/e41e137deae1e3fde5df93b1c114fc595043b875/openpype/hosts/maya/plugins/create/create_vrayscene.py#L42-L68) creator is the same as in Maya's [`CreateRender`](https://github.com/pypeclub/OpenPype/blob/e41e137deae1e3fde5df93b1c114fc595043b875/openpype/hosts/maya/plugins/create/create_render.py#L103-L140).

What's the part that is wrong about these? Both creators work on my end so I'm not sure what might be wrong. Are both wrong?

I've noticed there's also some similar logic here:
- [`openpype/modules/default_modules/deadline/plugins/publish/collect_default_deadline_server.py`](https://github.com/pypeclub/OpenPype/blob/develop/openpype/modules/default_modules/deadline/plugins/publish/collect_default_deadline_server.py)
- [`openpype/modules/default_modules/deadline/plugins/publish/collect_deadline_server_from_instance.py`](https://github.com/pypeclub/OpenPype/blob/develop/openpype/modules/default_modules/deadline/plugins/publish/collect_deadline_server_from_instance.py)

---

I'd however push towards moving the `get_deadline_urls` logic to the Deadline module and have all these do:
```python
deadline_settings = get_system_settings()["modules"]["deadline"]
if not deadline_settings["enabled"]:
    self.deadline_servers = {}

# pseudocode
import openpype_modules
openpype_modules.deadline.get_deadline_urls()
```
Or push it towards `openpype.lib` where currently also the `openpype.lib.abstract_submit_deadline` exists - but I feel that's worse than having it in the module.

Or if we know for sure that Deadline module is available (not necessarily enabled) then even the enabled check could be done in `openpype_modules.deadline.get_deadline_urls()`

**Comment by antirotor** [comment]
This was already solved by #2580 - it was using old env var to determine Deadline webservice URL. Use of Deadline code in OP is subject to refactoring as the same code is used on multiple places and so on.
